tests/basic.t

```diff
@@ -65,49 +65,94 @@ if [ $? -eq 1 ]; then
   TEST glusterd
 fi
 
-# Create gluster volume
+##### Create Gluster Volume Test Start #####
 TEST gluster vol create ${VOLNAME} ${HOST}:${BRKDIR} force
 
 # Start the volume
 TEST gluster vol start ${VOLNAME}
+##### End #####
 
 
-# Test gfapi access
+##### Gfapi Access Test Start #####
 TEST ./tests/gfapi-test ${VOLNAME} ${HOST}
+##### End #####
 
-# Start gluster-blockd.service
+
+##### Restart gluster-blockd.service Test Start #####
 systemctl daemon-reload
-TEST systemctl restart gluster-blockd.service
+TEST systemctl restart gluster-block-target.service gluster-blockd.service
 sleep 1;
+##### End #####
+
+
+##### Block 'create' and 'delete' Test Start #####
+# Simple block create/delete
+TEST gluster-block create ${VOLNAME}/${BLKNAME} ${HOST} 1MiB
+TEST gluster-block delete ${VOLNAME}/${BLKNAME}
+
+# Block create with 'auth' set/delete
+TEST gluster-block create ${VOLNAME}/${BLKNAME} auth enable ${HOST} 1MiB
+TEST gluster-block delete ${VOLNAME}/${BLKNAME}
+
+# Block create with 'ha' set/delete
+TEST gluster-block create ${VOLNAME}/${BLKNAME} ha 1 ${HOST} 1MiB
+TEST gluster-block delete ${VOLNAME}/${BLKNAME}
+
+# Block create with 'prealloc' set/delete
+TEST gluster-block create ${VOLNAME}/${BLKNAME} prealloc full ${HOST} 1MiB
+TEST gluster-block delete ${VOLNAME}/${BLKNAME}
+
+# Block create with 'ring-buffer' set/delete
+TEST gluster-block create ${VOLNAME}/${BLKNAME} ring-buffer 32 ${HOST} 1MiB
```

Comment:
@lxbsz This is failing, can you check:

TEST : gluster-block create hosting-volume/block-volume ring-buffer 32 192.168.124.147 1MiB
failed to configure on 192.168.124.147 configure failed                                                  
RESULT:FAIL                                                                                                
line 108 : NOT OK

Reply:
What's the kernel version ?
My local test is okay in RHEL and fedora 29.

Reply:
[root@Fedora-Server gluster-block]# uname -a
Linux Fedora-Server 5.0.3-200.fc29.x86_64 #1 SMP Tue Mar 19 15:07:58 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
[root@Fedora-Server gluster-block]# cat /etc/fedora-release 
Fedora release 29 (Twenty Nine)
[root@Fedora-Server gluster-block]# 



@lxbsz may be its coming from rtslib-v2.1.fb69 ? 

If it is passing for you ignore, I will debug the issue locally.

Thanks!

Reply:
yeah, sure please, and at the same time I will test more.